contracts/stake/src/state.rs

```diff
@@ -91,7 +102,7 @@ impl StakeState {
         let prev_stake = self.get_stake(&stake.keys().account).copied();
         let (loaded_stake, keys) = self.load_or_create_stake_mut(stake.keys());
 
-        if loaded_stake.amount.is_none() && value < MINIMUM_STAKE {
+        if loaded_stake.amount.is_none() && value < minimum_stake {
```

Comment:
```suggestion
        if loaded_stake.amount.is_none() && value < self.conf.minimum_stake {
```
Why not directly using the value instead of assigning it first?

Reply:
Because self is already borrowed mut, borrow checker was complaining